layers/generate_layers.py

```diff
@@ -173,6 +178,34 @@ def fixProps(inputProps):
 
     convertGeoJsonToPmtiles(f"{tmp}/mrn.geojson", "output/mrn.pmtiles")
 
+def makeNationalCycleNetwork():
+    tmp = "tmp_ncn"
+    ensureEmptyTempDirectoryExists(tmp)
+
```

Comment:
For posterity, a comment linking to https://data-sustrans-uk.opendata.arcgis.com/ useful. The arcgis URL isn't discoverable or could change with another version